README.md

```diff
@@ -3,6 +3,33 @@
 [![Backend CI](https://github.com/aggie-coding-club/Automatic-Aggie-Scheduler/workflows/Backend%20CI/badge.svg)](https://github.com/aggie-coding-club/rev-registration/actions?query=branch%3Amaster+workflow%3A%22Backend+CI%22)
 [![Frontend CI](https://github.com/aggie-coding-club/Automatic-Aggie-Scheduler/workflows/Frontend%20CI/badge.svg)](https://github.com/aggie-coding-club/rev-registration/actions?query=branch%3Amaster+workflow%3A%22Frontend+CI%22)
 
+![project-demo](assets/project-demo.gif)
+
+## Description
+
+Rev Registration is an automatic class scheduler for Texas A&M, aimed at easing the process
+of course registration by  generating compatible schedules given the courses a student would
+like to take and what preferences they have. Simply select a term, pick your courses, mark off when
+you're not available, and we'll generate schedules for you!
+
+For instance, imagine you've settled on 3 sections of a course you are fine with having and are having
+trouble finding a schedule without conflicts. Select all 3 under the Section customization level
+and let the schedule generator create schedules for you without time conflicts!
+
+**Features:**
+
+- Automatic schedule generation
+- Blocking off busy times (availabilities)
+- Grade distribution displaying
+- Displaying current seats remaining
+- Search by course number or title
+- Filter for only honors or online sections
+- Visual display of schedules
+- Persisting of sessions
+- Google Login
+- Saving, renaming, and deleting of generated schedules
```

Comment:
Technically, saved schedules aren't saved to the server yet, are they? Not a problem, since it's planned for launch, but I just want to make sure I'm caught up with where we are right now

Reply:
That's correct - I figured we'd be too lazy to add it to the README when we actually implement it in #267, so I just added it preemptively